scripts/ai-sdlc/verify-spec.mjs

```
@@ -0,0 +1,156 @@
+// required section headers survived before overwriting the spec.
+function sanitize(raw) {
+  let text = raw.trim();
+  if (text.startsWith('```')) {
+    text = text
+      .replace(/^```[a-z]*\n?/, '')
+      .replace(/\n?```$/, '')
+      .trim();
+  }
+  // Trim any leading preamble before the first section header.
+  const headerIdx = text.indexOf('## ');
+  if (headerIdx > 0) text = text.slice(headerIdx);
```

ESLint

[error] Expected { after 'if' condition.

(curly)